modules/network-observability-flowmetrics-charts.adoc

```diff
@@ -0,0 +1,102 @@
+// Module included in the following assemblies:
+//
+// network_observability/metrics-alerts-dashboards.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-custom-charts-flowmetrics_{context}"]
+= Configuring custom charts using FlowMetrics API
+You can generate charts for dashboards in the {product-title} web console, which you can view as an administrator in the *Dashboard* menu by defining the `charts` section of the `FlowMetric` resource.
+
+.Procedure
+. In the web console, navigate to *Operators* -> *Installed Operators*.
+. Under the *Provided APIs* heading for the *NetObserv Operator*, select *FlowMetrics*.
+. Use the *Project:*  dropdown list to select the project of the Network Observability Operator instance. 
+. Click *Create FlowMetric*.
+. Here are some examples for configuring the `FlowMetric` resource: 
+
+.Chart for tracking ingress bytes received from cluster external sources
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: flows.netobserv.io/v1alpha1
+kind: FlowMetric
+metadata:
+  name: flowmetric-cluster-external-ingress-traffic
+  namespace: netobserv   <1>
+# ...
+  charts:
+  - dashboardName: Main
+    title: External ingress traffic
+    unit: Bps
+    type: SingleStat
+    queries:
+    - promQL: "sum(rate($METRIC[2m]))"
+      legend: ""
+  - dashboardName: Main
+    sectionName: External
+    title: Top external ingress traffic per workload
+    unit: Bps
+    type: StackArea
+    queries:
+    - promQL: "sum(rate($METRIC{DstK8S_Namespace!=\"\"}[2m])) by (DstK8S_Namespace, DstK8S_OwnerName)"
+      legend: "{{DstK8S_Namespace}} / {{DstK8S_OwnerName}}"
+# ...
+----
+<1> The same namespace, netobserv, is used across all components. You can optionally use a different namespace for the different components. It is important to keep namespace names consistent across all components.
+
+This example creates two panels:
```

Comment:
```suggestion
This example creates two panels under existing NetObserv / Main dashboard:
```
we should also let users know, optionally they can configure under a new dashboard of their own and those charts would appear there.

Reply:
Ok I added info to both examples, so there are two separate verification steps that are in each collapsible sample block:

-https://github.com/openshift/openshift-docs/pull/74412/files#diff-4a2491a61cc9a63333a7315ba8f70962cc8e3baf7c29e141719ee3e5cb5e3140R49
-https://github.com/openshift/openshift-docs/pull/74412/files#diff-4a2491a61cc9a63333a7315ba8f70962cc8e3baf7c29e141719ee3e5cb5e3140R104

Reply:
One question I have about creating a dashboards with a custom name, we don't currently tell users how to do create a dashboard with their own name, as far as I am aware. I feel like we would need to say more about this, like if you're within the netobserv namespace, you can create a dashboard that will be prefixed with *Netobserv / <your_dashboard> and how exactly to do it.

Reply:
yes sounds good, I guess you can simply annotate the `dashboardName` in YAML to tell that using a different name would create a new dashboard

Reply:
I like that suggestion a lot :) Thanks!

Reply:
Ok I added annotations to the `dashboardName` in the YAML.